src/activities/quizzes/QuizEntity.js

```diff
@@ -197,6 +197,78 @@ export class QuizEntity extends Entity {
 		return entity && entity.hasClass(Classes.quizzes.checked);
 	}
 
+	/**
+	 * @returns {string} Quiz discription in plaintext (HTML stripped)
+	 */
+	descriptionPlaintext() {
+		const descriptionEntity = this._getDescriptionEntity();
+		return descriptionEntity
+			&& descriptionEntity.properties
+			&& descriptionEntity.properties.text;
+	}
+
+	/**
+	 * @returns {string} Quiz description in HTML
+	 */
+	descriptionHtml() {
+		const descriptionEntity = this._getDescriptionEntity();
+		return descriptionEntity
+			&& descriptionEntity.properties
+			&& descriptionEntity.properties.html;
+	}
+
+	/**
+	 * @returns {string} Quiz description formatted to be used with a d2l-html-editor
+	 */
+	descriptionEditorHtml() {
+		const descriptionEntity = this._getDescriptionEntity();
+		if (!descriptionEntity) {
+			return;
+		}
+
+		const updateDescriptionAction = descriptionEntity.getActionByName(Actions.quizzes.updateDescription);
+		return updateDescriptionAction
+			&& updateDescriptionAction.hasFieldByName('description')
```

Comment:
Do we want to make "description" a constant?

Reply:
That's a good point, maybe all the field names should be defined as constants? Makes sense for 'description' as it's used a few times, but it might be cleaner to be consistent with all field names. Maybe a clean-up task for a separate PR.